cleanup_everything_bazel_manually.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
@eslutsky where do we call this script from?

Reply:
we are using this script to create the environment locally (not from our  ci), 
we can try merging cleanup_everything_bazel_manually.sh and cleanup_everything_bazel.sh together